kmultiversx/src/kmultiversx/kdist/plugin.py

```diff
@@ -78,21 +79,12 @@ def deps(self) -> tuple[str, str]:
 
 
 def llvm_target(main_file_name: str, main_module: str, syntax_module: str) -> KompileTarget:
-    return KompileTarget(
-        lambda src_dir, plugin_dir: {
-            'backend': KompileBackend.LLVM,
-            'main_file': src_dir / 'mx-semantics' / main_file_name,
-            'main_module': main_module,
-            'syntax_module': syntax_module,
-            'include_dirs': [src_dir],
-            'md_selector': 'k',
-            'hook_namespaces': ['KRYPTO'],
-            'opt_level': 2,
-            'ccopts': [
+    def ccopts(plugin_dir: Path) -> list[str]:
+        is_linux = sys.platform.startswith('linux')
```

Comment:
The `startswith` usage is recommended by the official docs, but I will inline it into the conditional.

Reply:
Right, but it also contains this part:

> _Changed in version 3.3_: On Linux, `sys.platform` doesn’t contain the major version anymore. It is always `'linux'`, instead of `'linux2'` or `'linux3'`.

Reply:
Good catch, thanks!